### PR TITLE
BB-28 Use the data in the accounts component

### PR DESCRIPTION
**Pull request by jbarriguete86**
add a utilities.js with a formula to transform the data in the accounts to USD, also the account component is now using the firebase database to populate the information

### CODE REVIEW

**Review by shantdashjian — COMMENTED** [review]
Why map over accountData? It's an object that has the data of the 3 types of accounts, right? Instead of:
```
const accountList = accountData.map((data) => {
    let accountsArr = []
    accountsArr.push(data.Main_Account,data.Savings,data.Expenses)

    return (accountsArr.map(account=>(
```
    
    Why not 
```
    const accountsArr = [accountData.Main_Account,data.Savings,data.Expenses]

    return (accountsArr.map(account=>(
```

**Comment by jbarriguete86** [comment]
> Why map over accountData? It's an object that has the data of the 3 types of accounts, right? Instead of:
> 
> ```
> const accountList = accountData.map((data) => {
>     let accountsArr = []
>     accountsArr.push(data.Main_Account,data.Savings,data.Expenses)
> 
>     return (accountsArr.map(account=>(
> ```
> 
> ```
> Why not 
> ```
> 
> ```
>     const accountsArr = [accountData.Main_Account,data.Savings,data.Expenses]
> 
>     return (accountsArr.map(account=>(
> ```

> ```
>something that might work is:
> ```
> ```
>const accountsArr= [accountData[0].Main_Account, accountData[0].Savings,accountData[0].Expenses]
 >const accountList = accountsArr.map((account) => {
 >  return (
>    <AccountCard
>      key={account.id}
>      title={account.title}
>      balance={setUsdCurrency(account.balance)}
>    />
>    )
>  });
> ```

File: src/pages/HomePage.jsx
Comment (shantdashjian) [inline src/pages/HomePage.jsx:1]: If there is no account data, I would like to see a message like "No accounts available at this time".

```
{accountData ? accountList : "No accounts are available at this time"}
```

**Review by shantdashjian — COMMENTED** [review]
Is there an account.title in the data?

**Comment by jbarriguete86** [comment]
> Is there an account.title in the data?

There is an account.title in the firebase database

**Review by shantdashjian — APPROVED** [review]
All looks good. Closing.

**Pull request merged**